src/wrappers/themis/jsthemis/test/test.js

```diff
@@ -1,4 +1,4 @@
-var addon = require('jsthemis');
+var addon = require('..');
```

Comment:
as I remember, the main idea of `tests/<wrapper>/` folder is to have one place of all tests which allow to test installed wrapper to system. And tests inside package's folders used to test their source code and functionality. 
`tests/jsthemis/test.js` and `src/wrappers/themis/jsthemis/test/test.js` have different purposes. First to check the installed package, second to test code from sources.

Reply:
I think we can get away with not having a standalone test suite, given that JsThemis normally will not be installed globally. However, if we package the tests along with the source code then it's possible to test JsThemis by going into its directory in node_modules and running `npm install && npm test` from there, without having to check out the repository.

It seems that npm does not provide a way to install a package and run its tests. I guess it is expected that developers do all the testing before publishing it and the users don't need to do it.

Reply:
I think we should pack tests with source codes anyway. It's useful when user will install via npm and will try to change/patch something in-place and test it.